src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,50 @@
+/* eslint-disable */
 import React from 'react';
 import './CurrentUser.scss';
+import { getUser } from '../../api/api';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends React.Component {
+  state = {
+    user: [],
+  };
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+  async componentDidMount() {
+    const user = await getUser(this.props.userId);
+    this.setState({ user });
```

Comment:
You are duplicating this code here and in componentDidUpdate.
Let's move it to a separate function and call it twice